### PR TITLE
Return a dict of image_id and device_id instead of just image_id

### PR DESCRIPTION
**Pull request by turbokongen**
Return a dict with image_id as key and device_id as value.
Usecase is to accsess the device_id of the cam from applications.

### CODE REVIEW

File: verisure/devices/smartcam.py [hunk 1]
```diff
@@ -42,10 +42,13 @@ def get_imagelist(self):
         image_series = status['imageSeries']
         image_data_list = [li['images'] for li in image_series]
         n = len(image_data_list)
-        image_ids = []
+        image_ids = {}
         for i in range(0, n):
-            image_id = [li['id'] for li in image_data_list[i]]
-            image_ids.append(image_id)
+            lookup_image_id = [li['id'] for li in image_data_list[i]]
+            image_id = str(''.join(lookup_image_id))
```
Comment (persandstrom) [inline verisure/devices/smartcam.py]: Is there always only one lookup_image_id and one lookup_device_id for each iteration? 
It looks a bit strange that image_id is the concatenation of all lookup_image_ids.

File: verisure/devices/smartcam.py [hunk 2]
```diff
@@ -42,10 +42,13 @@ def get_imagelist(self):
...
+            lookup_device_id = [li['deviceLabel'] for li in image_data_list[i]]
+            device_id = str(''.join(lookup_device_id))
+            image_ids[image_id] = device_id
         return image_ids
```
Comment (persandstrom) [inline verisure/devices/smartcam.py]: would it be possible to return a dict formatted as bellow?  
```
{ 
  'device1': ['image1_1', 'image1_2' ...],
  'device2': ['image2_1', 'image2_2' ...],
}
```

Comment (turbokongen) [inline verisure/devices/smartcam.py]: Done :D

File: verisure/devices/smartcam.py (hunk 1 above)
Comment (persandstrom) [inline verisure/devices/smartcam.py]: This structure is a bit complicated :)
could you try something like this instead:
```
for image_data in image_data_list:  
    image_id = image_data[0]['id']
    device_id = image_data[0]['deviceLabel']
    image_ids.setdefault(device_id, []).append(image_id)
```

Comment (turbokongen) [inline verisure/devices/smartcam.py]: That works :D

**Comment by persandstrom** [comment]
Looks good, i'll release it

**Pull request merged**